tasks/Task.cpp

```diff
@@ -4,6 +4,51 @@
 
 using namespace imu_kvh_1750_mixed;
 
+int AuxImuDriver::extractPacket(uint8_t const *buffer, size_t buffer_size) const {
+
+	static bool found_start = false;
```

Comment:
Like ... no. Not ever. static local variables ? NO.

Really simply re-parse the whole buffer at each call. You can't assume to know what the caller does with the buffer.

@joaobrittoneto: could you give him a hand here ?